src/Components/DnDLayout/GridTile.tsx

```diff
@@ -62,6 +65,14 @@ const GridTile = ({ widgetType, isDragging, setIsDragging, setWidgetAttribute, w
     return null;
   }
 
+  const widgetLink = (href: string) => {
+    if (href.includes('https://')) {
+      return href;
+    } else {
+      return `${window.location.origin}${chrome.isBeta() ? '/preview' : ''}${href}`;
```

Comment:
You don't have to add origin or the preview partial to router links. That is handled via the component.